resumablejs/resumable.d.ts

```diff
@@ -0,0 +1,374 @@
+// Type definitions for Resumable.js v1.0.2
+// Project: https://github.com/23/resumable.js
+// Definitions by: Daniel McAssey <https://github.com/DanielMcAssey>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace Resumable {
```

Comment:
I am asuming that Resumable is going to be only used as an npm pacakge, is this correct?

if so, then you do not need to put this name in the global namespace. Instead define these inside the module declaration, e.g.:

```ts

declare module 'resumablejs' {
   namespace Resumable {
    interface ConfigurationHash { ... }
    class Resumable { ... }
  }
  export = Resumable;
}
```